app/src/main/java/tech/relaycorp/letro/awala/AwalaManager.kt

```diff
@@ -225,11 +219,11 @@ class AwalaManagerImpl @Inject constructor(
             }
             isReceivingMessages = true
 
-            Log.i(TAG, "start receiving messages...")
-            GatewayClient.receiveMessages().collect { message ->
-                Log.i(TAG, "Receive message: ${message.type}: ($message)")
+            logger.i(TAG, "start receiving messages...")
+            awala.receiveMessages().collect { message ->
+                logger.i(TAG, "Receive message: ${message.type}: ($message)")
```

Comment:
Same thing here with `$message` -- I think we should skip the contents of the message, but the sender and recipient may be useful

Reply:
Yes, nice catch! Anyway, this info is useless in logs